### PR TITLE
3437 checkpoints for carto at scale

### PR DESCRIPTION
**Pull request by liu-zhipeng**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/agents/cartographer/poller/src/lib/operations/transfers.ts [hunk 1]
```diff
@@ -12,23 +12,23 @@ import {
 import { getContext } from "../../shared";
 
 const getMaxNonce = (transfers: DestinationTransfer[] | XTransfer[]): number => {
-  return transfers.length == 0 ? 0 : Math.max(...transfers.map((transfer) => transfer.xparams.nonce ?? 0));
+  return transfers.length == 0 ? 0 : Math.max(...transfers.map((transfer) => transfer.origin?.xcall.txNonce ?? 0));
```
Comment (rhlsthrm) [inline packages/agents/cartographer/poller/src/lib/operations/transfers.ts:15]: Since xcall already has a nonce I don't think we need this, right?

Comment (liu-zhipeng) [reply packages/agents/cartographer/poller/src/lib/operations/transfers.ts:15]: Yes, right. but I was going to process them consistently

Comment (rhlsthrm) [reply packages/agents/cartographer/poller/src/lib/operations/transfers.ts:15]: Right. But having nonce already makes this unnecessary, so I think it's better to just remove it.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]
Remove the xcall processing.

**Review by rhlsthrm — DISMISSED** [review]

**Review by preethamr — DISMISSED** [review]

**Review by wanglonghong — APPROVED** [review]

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**